web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -40,11 +46,11 @@ consider when writing a PDEP are:
 
 ### PDEP authors
 
-Anyone can propose a PDEP, but in most cases developers of pandas itself and related
-projects are expected to author PDEPs. If you are unsure if you should be opening
-an issue or creating a PDEP, it's probably safe to start by
-[opening an issue](https://github.com/pandas-dev/pandas/issues/new/choose), which can
-be eventually moved to a PDEP.
+Anyone can propose a PDEP, but core members need to sponsor a proposal made by non-core
+contributors. To submit a PDEP as a community member, please propose the PDEP concept on
+[an issue](https://github.com/pandas-dev/pandas/issues/new/choose), and find a pandas team
+member to collaborate with. They can co-author the PDEP with you and submit it to the PDEPs
```

Comment:
I think there are a few different opinions on this, and it might be worth to see how many people prefer it this way or that before deciding to make a change.

Reply:
In addition to adding the team so non-core contributors can find someone to help sponsor the PDEP, maybe noting the Slack group might be a good way to also find a sponsor?

Reply:
On the "They can co-author the PDEP with you and submit it to the PDEPs repository" part, that sounds as it is the sponsor/co-author that needs to do that. But if the core-team member is just sponsoring (guiding through the process), I think the non-core team member can certainly open a PR themselves?

Reply:
Is there a particular problem we're trying to solve with the requirement for a core dev sponsor of PDEPs? Feels to me just extra bureaucracy, and core devs can already close low quality PDEPs as invalid, which would be equivalent.

Reply:
> Is there a particular problem we're trying to solve with the requirement for a core dev sponsor of PDEPs? Feels to me just extra bureaucracy, and core devs can already close low quality PDEPs as invalid, which would be equivalent.

Copying here what I posted in the non-line comment about general summary of the changes: IIRC the reason for this was two-fold: to have _some_ form of quality assurance (to not be "spammed" by many drive-by PDEPs that might not need to be a PDEP, although this might not happen quickly in practice) and to help the non-core-team submitter (the sponsor would be a core-team member who takes on the responsibility of guiding the PDEP author through the process).

The "sponsor" term is inspired on the PEP process (https://peps.python.org/pep-0001/#submitting-a-pep, _"The sponsor’s job is to provide guidance to the PEP author to help them through the logistics of the PEP process (somewhat acting like a mentor)."_), but we should probably explain that in a bit more detail in the text here as well, if we keep this.

Reply:
indeed. The concept of a mentor makes more sense than using the term sponsor suggests.

Reply:
I still think we're fixing a problem we don't have, which doesn't seem like a good idea in general. Probably when we do have too many low quality PDEPs we will want to revisit this anyway (and I feel just closing them as invalid may make more sense).

If someone from the Arrow, Dask, scikit-learn, Intel... Wants to propose a PDEP, I don't know how requiring them to have a sponsor or a mentor is helpful. I really like the idea of offering a mentor for PDEPs coming from less experienced people. But I don't see how this is helpful (if you want to spam with a low quality PDEP, you can dl it anyway whatever we say here), and I do see how this can be harmful.

Reply:
Thoughts on Irv's proposal to rename it (sponsor -> advisor)? 

Personally, I am also fine to just leave out this addition of the sponsor mention. It's not really related to the other changes to clarifying scope etc, and we can always add it later if we want.

Reply:
I don't like the "sponsor required" idea, happy with "advisor provided" one or just leave this out, both are fine.

Reply:
I used @Dr-Irv's suggestion for now. I'd happy to get additional feedback on this in the future - I think it's an important point to get right.